apps/web/src/pages/block/[id].tsx

```diff
@@ -78,7 +79,15 @@ const Block: NextPage = function () {
 
     detailsFields = [
       { name: "Block Height", value: blockData.number },
-      { name: "Hash", value: blockData.hash },
+      {
+        name: "Hash",
+        value: (
+          <div className="flex items-center gap-2">
```

Comment:
I've noticed this structure is repeated also in https://github.com/Blobscan/blobscan/pull/478/files#r1713044581 . I think we could move it to the commonplace to avoid further misalignments